### PR TITLE
add court allocation back into the build

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

Fix bug where court allocation was removed from the submission step.

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Manual and units

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/steps/check-your-answers/index.test.js [hunk 1]
```diff
@@ -766,6 +770,15 @@ describe(modulePath, () => {
       });
     });
 
+    it('allocates a court on submission', done => {
+      // Act.
+      testCustom(done, agent, underTest, [], () => {
+        // Assert.
+        expect(courtsAllocationSpy.calledOnce)
```
Comment (Rob-Doherty) [inline app/steps/check-your-answers/index.test.js]: Can you add a check that the result of the `courtsAllocation.allocateCourt()` call has been added to the session?
When I change `req.session.courts = courtsAllocation.allocateCourt();` to just `courtsAllocation.allocateCourt();` the unit test still passes, so could be a bit more robust

Comment (Rob-Doherty) [reply app/steps/check-your-answers/index.test.js]: Looks like the old unit test covered the session content and Google Analytics as well, so maybe easiest to just use that? https://github.com/hmcts/div-petitioner-frontend/blob/court-allocation-DIV-2795/app/steps/submit/index.test.js#L178-L220

Comment (dtaylor7) [reply app/steps/check-your-answers/index.test.js]: no need its covered in the test of courtsAllocation file `https://github.com/hmcts/div-petitioner-frontend/blob/master/app/services/courtsAllocation.test.js#L23`

Comment (dtaylor7) [reply app/steps/check-your-answers/index.test.js]: done

**Comment by nsingamsetty** [comment]
This is tested from Divorce prospective and could not verify this in CCD AAT as the Environment is down but when we submit the case this is assigned to respective RDC. Please refer the screenshots for the DIV-2857.Consulted the same with Rhys and he is happy hence merging the code.

**Pull request merged**